test/common/testlib.py

```diff
@@ -93,6 +93,17 @@
 opts.jobs = 1
 opts.fetch = True
 
+layout_scenarios = {
+    "desktop": {
+        "shell_size": (1920, 1200),
+        "content_size": (1680, 1130)
+    },
+    "mobile": {
+        "shell_size": (414, 1920),
+        "content_size": (414, 1856)
+    }
+}
```

Comment:
I'm not sure this belongs in `common/`.  The choice of which sizes to test is probably reasonably per-project, no?

I'd expect this to live in a file in the specific repository, or in the reference images repository.

Reply:
Done, it's in `test/browser-layouts`.

Reply:
There is a reasonable default now, so that projects don't *have* to define their own scenarios -- and I expect most dont', and we should not even recommend it. There is a reason for why we pick these scenarios, and they are not exactly trivial or arbitrary. I like the current approach with offering the possibility of overrides, but not making too much fuss about it.